### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by elmar8287**
- Create a directory called components.
- Inside components, create a new Calculator.js file.
- Give style as in requirement.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,5 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => <Calculator />;
```
Comment (AshakaE) [inline src/App.js]: The `App.js` component should be class-based.

- [x] Please change this to a class based component

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        <button type="button">2</button>
+        <button type="button">3</button>
+        <button type="button" className="orange">+</button>
+        <button type="button" className="zero">0</button>
+        <button type="button">.</button>
+        <button type="button" classNameName="orange">=</button>
+      </section>
+    </div>
+  );
+};
```
Comment (AshakaE) [inline src/components/Calculator.js]: The `Calculator.js` component should be class-based

- [x] Please change this to a class based component

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @elmar8287  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!

#  Required Changes ♻️ 🚧
_Check the comments under the review._

- [x] Please update your readme to do away with the boilerplate readme that comes with `npx create-react-app app-name`

- [x] Please try to match the given design

Design |
------------ |
image |




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (elmar8287) [reply src/App.js]: It is done. Thanks for note

File: src/components/Calculator.js (hunk 2 above)
Comment (elmar8287) [reply src/components/Calculator.js]: It is done. Thanks for note

**Review by george-shammar — APPROVED** [review]
Hi @elmar8287 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

:heavy_check_mark: Thanks for making the required changes . :+1: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by elmar8287** [comment]
Many thanks for review 👍

**Pull request merged**